tests/actions/test_core_actions.py

```diff
@@ -566,44 +566,26 @@ def test_SlackNotificationAction_run(self, checkpoint_result: CheckpointResult):
             json={
                 "blocks": [
                     {"text": {"text": mock.ANY, "type": "plain_text"}, "type": "header"},
-                    {"type": "divider"},
                     {
+                        "type": "section",
+                        "text": {"type": "plain_text", "text": "Runtime: 2024/04/01 08:51 PM"},
```

Comment:
Could we get a test with multiple assets? Additionally, could you add a screenshot of what that renders like?

Reply:
added another test in `test_core_actions.py` to cover this scenario